### PR TITLE
Implemented heatmap based on Smash Magazine landing page.

### PR DESCRIPTION
**Pull request by pointerish**
The heatmap is in the scale of grey. This was done for the Microverse UI/UX module.

### CODE REVIEW

**Review by blackpintz — CHANGES_REQUESTED** [review]
# Status: Changes Required

## Hello team,

**Good job on your project. However, there a few changes required.**

1). **Please use `<article>` tags for all your articles.**

2). **Ensure that articles on the same row share an equal portion of the page total width. For example, in the screenshot below, you gave each article a width of 40% instead of 50%. Please adjust that.**

design_clone

3). **Images used on your page should also be grayscale(white, black or any shade of grey). Hint: Image filters can help you with this.**

4). **Change all the blue links to grayscale colors.**

5). **Convert your footer list to links. It is a requirement that the footer should have centered links.**

6). **Add a Getting started section, on your readme file, with instructions on how one can set up your project locally and the necessary tools required. You can use this readme [template](https://github.com/microverseinc/readme-template) as reference.**

7). **Remove the page horizontal scroll.**

### Happy coding 😃.

**PS: You can reach me on slack(Rose Wanjohi)**

**Comment by pointerish** [comment]
Hi!

See the screenshot below:

expected50

This is with the articles taking 50% of the width as requested by the previous TSE. Personally this seems to attempt against aesthetics but if that is expected then that's okay.

We previously had 40% and the articles looked like floating blocks with a ligther background. We preferred that, but we understand these projects are not based upon our personal preferences.

Thank you for all your help!

**Review by acushlakoncept — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

### Hi, @pointerish and @imran5644,

Both of you have done well on this project.

Hello team, don't forget the goal for this project is to start training yourself to think in terms of visual hierarchy, typography, and design principles. Meaning the weight of each section in terms of colors, which catches your attention first.

If you get this, you will understand what the previous TSE meant, here is a [chrome extension](https://chrome.google.com/webstore/detail/grayscale-the-web-save-si/mblmpdpfppogibmoobibfannckeeleag?hl=en) that will make your job simple, more emphasis is how you represent these visual hierarchies.

Note: Again! emphasis is on the visual hierarchy! See the color variation on the screenshot below:
Screen Shot 2020-10-02 at 3 19 23 AM

Make these changes to your design and you should be good to go.

### Cheers and Happy coding! 💻 👏

(Feel free to contact me on slack(@acushla), if you have any additional question)
Uduak Essien
[Twitter](https://twitter.com/acushlakoncept) | [LinkedIn](https://www.linkedin.com/in/acushlakoncept/)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,46 @@
...
+
+A Smashing Magazine HeatMap built for Microverse's UI/UX module.
+
+![screenshot](screenshot.png)
+
+
+## Built With
+
+- HTML & CSS
+
```
Comment (javitocor) [inline README.md:13]: Make each technology in a different line

File: README.md [hunk 2]
```diff
@@ -1,2 +1,46 @@
...
+##  Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+##  Getting Started
+
+To run the code simply download the repository by either cloning it or 
+downloading it as Zip file. Then use the old trustable browser to open the HTML file.
+
```
Comment (javitocor) [inline README.md:39]: This section should before the author's section so the users do not need to scroll down

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Great job so far, team! 👏 
You did well implementing the changes required by the previous TSE 👍 
You are almost there! 😃 

Just a few adjustments and ready to go! ⏩ 
Kindly, check my comments below 👇  apply the changes and resubmit your project for a new code review

Good luck!

## Changes
- Kindly add more links to your footer, we are trying to make clear the hierarchy and importance of each element in the website. Check the difference between the original and your project 👇 
Capture1
Capture
- In the community links section, the layout is not the same as the original website, which makes that the importance of every elements is different. Kindly, modify the layout and apply the corresponding colour.
Capture3
Capture2
- Kindly add the 'seriously, ted?' banner in the main section. It modifies the hierarchy
Capture4
 of the site

**Review by blackpintz — CHANGES_REQUESTED** [review]
# Status: Changes Required

## Hello team,

**Your project is good but there is still a horizontal scroll on your page. Please fix that and send for another review.**

### Happy coding 😃.

**ps: You can reach me on slack(Rose Wanjohi).**

**Comment by pointerish** [comment]
Hi TSEs,

We have updated the screenshot and we have also removed the unneeded horizontal scrolling. Thanks for the patience and for the feedback!

**Review by blackpintz — APPROVED** [review]
# Status: Approved.

**Congratulations team and all the best in your next project 😃.**

**Pull request merged**